### PR TITLE
Things to make it work

### PR DESCRIPTION
**Pull request by ItsMattG**

### CODE REVIEW

File: src/styles/BorrowingCalculatorElements.tsx [hunk 1]
```diff
@@ -153,6 +167,14 @@ export const BoxOne = styled.div`
                 font-size: 13px;
                 line-height: 15px;
                 color: #575757;
+
+                margin-left: 7px;
+                appearance: none;
+                -moz-appearance: none;
+                -webkit-appearance: none;
+                background-image: url("https://i.imgur.com/5v17RRC.png");
```
Comment (Xenarc) [inline src/styles/BorrowingCalculatorElements.tsx:175]: With these `background-image:` you can actually use `background-image: ${MyImage};` and `import MyImage from '../assets/myimage.jpg';`

File: src/views/login.tsx [hunk 2]
```diff
@@ -1,35 +1,220 @@
...
+        };
+        return {
+            handleSubmit,
+            handleInputChange,
+            inputs,
+        };
+    };
+
+    const validate = (inputs: any) => {
+        const cond = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(
```
Comment (Xenarc) [inline src/views/login.tsx:75]: This is kinda scarey but tbh I don't care anymore about email validation using regex

File: src/views/login.tsx [hunk 3]
```diff
@@ -1,35 +1,220 @@
...
+                                    <ErrorMessage>
+                                        {errors.password}
+                                    </ErrorMessage>
+                                </p>
+                            )}
+                        </div>
+                        <Button
+                            onClick={() =>
+                                getSignIn({
+                                    variables: {
```
Comment (Xenarc) [inline src/views/login.tsx:193]: TODO: fix

File: src/views/signup.tsx [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+    const [inputs, setInputs] = useState<{ email?: string; password?: string }>(
+        {}
+    );
+
+    const useForm = React.useCallback(
+        (initialValues: any) => {
+            const handleSubmit = (e: any) => {
+                if (e) {
+                    e.preventDefault();
+                }
```
Comment (Xenarc) [inline src/views/signup.tsx:37]: Consider `e?.preventDefault();`

**Review by Xenarc — COMMENTED** [review]
Great work!

File: src/styles/BorrowingCalculatorElements.tsx (hunk 1 above)
Comment (ItsMattG) [reply src/styles/BorrowingCalculatorElements.tsx:175]: I tried that!!!!! But it didn't work :( so I resorted to uploading it to imgur haha!

Comment (ItsMattG) [reply src/styles/BorrowingCalculatorElements.tsx:175]: Acutally I don't think I used the "$" sign... I just did `{MyImage}`

File: src/views/login.tsx (hunk 2 above)
Comment (ItsMattG) [reply src/views/login.tsx:75]: I'll remove it :D I don't think it's neccessary anyway with this project.

File: src/views/login.tsx (hunk 3 above)
Comment (ItsMattG) [reply src/views/login.tsx:193]: Yep yep amen!

File: src/views/signup.tsx (hunk 4 above)
Comment (ItsMattG) [reply src/views/signup.tsx:37]: Hmm, what is the difference between the two?

**Pull request merged**